Source/FieldSolver/WarpX_QED_Field_Pushers.cpp

```diff
@@ -0,0 +1,235 @@
+            const Box& tby  = mfi.tilebox(By_nodal_flag);
+            const Box& tbz  = mfi.tilebox(Bz_nodal_flag);
+            auto const& pml_Bxfab = pml_B[0]->array(mfi);
+            auto const& pml_Byfab = pml_B[1]->array(mfi);
+            auto const& pml_Bzfab = pml_B[2]->array(mfi);
+            auto const& pml_Exfab = pml_E[0]->array(mfi);
+            auto const& pml_Eyfab = pml_E[1]->array(mfi);
+            auto const& pml_Ezfab = pml_E[2]->array(mfi);
+            if (WarpX::maxwell_fdtd_solver_id == 0) {
+               amrex::ParallelFor(tbx, tby, tbz,
+               [=] AMREX_GPU_DEVICE (int i, int j, int k) {
+                   warpx_push_pml_bx_yee(i,j,k,pml_Bxfab,pml_Eyfab,pml_Ezfab,
+                                        dtsdy,dtsdz);
+               },
+               [=] AMREX_GPU_DEVICE (int i, int j, int k) {
+                   warpx_push_pml_by_yee(i,j,k,pml_Byfab,pml_Exfab,pml_Ezfab,
+                                         dtsdx,dtsdz);
+               },
+               [=] AMREX_GPU_DEVICE (int i, int j, int k) {
+                   warpx_push_pml_bz_yee(i,j,k,pml_Bzfab,pml_Exfab,pml_Eyfab,
+                                        dtsdx,dtsdy);
+               });
+            }  else if (WarpX::maxwell_fdtd_solver_id == 1) {
+               Real betaxy, betaxz, betayx, betayz, betazx, betazy;
+               Real gammax, gammay, gammaz;
+               Real alphax, alphay, alphaz;
+               warpx_calculate_ckc_coefficients(dtsdx, dtsdy, dtsdz,
+                                                betaxy, betaxz, betayx, betayz,
+                                                betazx, betazy, gammax, gammay,
+                                                gammaz, alphax, alphay, alphaz);
+
+               amrex::ParallelFor(tbx, tby, tbz,
+               [=] AMREX_GPU_DEVICE (int i, int j, int k) {
+                   warpx_push_pml_bx_ckc(i,j,k,pml_Bxfab,pml_Eyfab,pml_Ezfab,
+                                         betaxy, betaxz, betayx, betayz,
+                                         betazx, betazy, gammax, gammay,
+                                         gammaz, alphax, alphay, alphaz);
+               },
+               [=] AMREX_GPU_DEVICE (int i, int j, int k) {
+                   warpx_push_pml_by_ckc(i,j,k,pml_Byfab,pml_Exfab,pml_Ezfab,
+                                         betaxy, betaxz, betayx, betayz,
+                                         betazx, betazy, gammax, gammay,
+                                         gammaz, alphax, alphay, alphaz);
+               },
+               [=] AMREX_GPU_DEVICE (int i, int j, int k) {
+                   warpx_push_pml_bz_ckc(i,j,k,pml_Bzfab,pml_Exfab,pml_Eyfab,
+                                         betaxy, betaxz, betayx, betayz,
+                                         betazx, betazy, gammax, gammay,
+                                         gammaz, alphax, alphay, alphaz);
+               });
+
+            }
+        }
+    }
+*/
```

Comment:
?

Reply:
Ahh that was from when I had to update my code post-someone else's PR. I held onto the original code in case I broke something. I'll delete it now.